docs/installation/migrate.rst

```diff
@@ -47,43 +47,64 @@ To migrate |sr| to |ccloud|, follow these steps:
     .. code:: bash
 
         curl -X PUT -H "Content-Type: application/json" -H "Authorization: Basic dGVuYW50Mi1rZXk6bm9oYXNo" "http://destregistry:8081/mode" --data '{"mode": "IMPORT"}'
+        
+    .. tip:: In the above command, authorization is provided as a Base64 encoded API key and API secret, which is recommended. 
+    
+    Here is an example showing one way to accomplish Base64 encoding and use environment variables to simplify the command to set |sr| mode. 
+    
+    - Pipe the key pair through a utility called `base64` and assign the result to a variable `BASIC_AUTH`.
+    - Assign the Schema Registry URL to another variable `CCSR_URL`.
+    - Finally, use both the variables in the `curl -X PUT -H ` command to set the destination Schema Registry to IMPORT mode.
+    
+    .. code:: bash
+
+        export BASIC_AUTH=$(echo -n 2GXRGGJJ2BNUNPFE:uZoTd8z/mXWqYPLEU++NTYjOOIZmd2e3BC8btZX4nyI4RNFbItwBSTekT1ntz511 | base64)
+
+        export CCSR_URL=https://psrc-e8j00.us-west-2.aws.devel.cpdev.cloud
+
+        curl -v -X PUT -H "Content-Type: application/vnd.schemaregistry.v1+json" -H "Authorization: Basic ${BASIC_AUTH}" ${CCSR_URL}/mode --data '{"mode": "IMPORT"}'
 
-#.  Configure Replicator with Schema Registry information.
 
+#.  Configure Replicator with Schema Registry information.
 
     :: 
     
+        // Schema Registry migration topics
+        topic.whitelist=mytopic1,mytopic2,_schemas
```

Comment:
@rayokota, @rhauch , @NathanNam  : my comment here is a dupe of the one in the 1-pager, but the condensed version is that I thought we were going to recommend a separate Replicator instance (without copying user topics like mytopic1, mytopic2) because it may need to be stopped separately, and schemas need to be copied before data is consumed.  Having it be a separate instance seems cleaner

Reply:
I agree with @ybyzek. While this set up works as well, but let's remove mytopic1, mytopic2 from the example.